assets/altis-seo.css

```diff
@@ -75,6 +86,18 @@
 	border-left-color: var( --altis-blue-darker-10 );
 }
 
+.assessment-results__mark:focus,
+.yoast_help.yoast-help-button:focus .yoast-help-icon::before,
+.yoast_help.yoast-help-link:focus .yoast-help-icon::before {
+	box-shadow: 0 0 0 1px #2041ab, 0 0 2px 1px rgba( 24, 48, 128, 0.8 );
```

Comment:
Not needed but fun fact, CSS variables can be partial too for example:

```css
:root {
  --altis-blue-rgb: 24, 48, 128;
  --altis-blue: rgb( var( --altis-blue ) );
}

.thing {
  box-shadow: 0 0 0 1px rgba( --altis-blue-rgb, 0.8 );
}
```

Reply:
When I was using CSS variables inside attributes that were compounded (e.g. `box-shadow` above), the colors didn't always work, so for those, as a rule, I defaulted to hex (or rgba) values. Might be a Firefox/browser compatibility thing.

That said, TIL you can use variables inside rgba values. 🤯